### PR TITLE
Update current release notes

### PR DESCRIPTION
**Pull request by jander-msft**

### CODE REVIEW

File: documentation/releaseNotes/releaseNotes.md [hunk 1]
```diff
@@ -3,4 +3,8 @@ Today we are releasing the 6.2.0 build of the `dotnet monitor` tool. This releas
 - Allow for pushing a message to a queue when writing to Azure egress (#163)
 - Allow for simplified process filter configuration (#636)
 - Allow `config show` command to list configuration sources (#277)
-- Added Collection Rule Defaults (#1595)
+- Added collection rule defaults (#1595)
+- Added collection rule templates (#1921)
+- Added CPU usage, GC heap size, and threadpool queue length triggers (#1911)
+- Show warning when using collection rules in `connect` mode (#1844)
+- Prevent auth configuration warnings when using `--no-auth` switch (#1845)
```
Comment (wiktork) [inline documentation/releaseNotes/releaseNotes.md:10]: I'm aiming to get the MSI work here too.

Comment (jander-msft) [reply documentation/releaseNotes/releaseNotes.md:10]: Sure, I'll add it in once you merge #1884

**Review by kelltrick — APPROVED** [review]

File: documentation/releaseNotes/releaseNotes.md [hunk 2]
```diff
@@ -3,4 +3,12 @@ Today we are releasing the 6.2.0 build of the `dotnet monitor` tool. This releas
 - Allow for pushing a message to a queue when writing to Azure egress (#163)
 - Allow for simplified process filter configuration (#636)
 - Allow `config show` command to list configuration sources (#277)
-- Added Collection Rule Defaults (#1595)
+- Added collection rule defaults (#1595)
+- Added collection rule templates (#1921)
+- Added CPU usage, GC heap size, and threadpool queue length triggers (#1911)
+- Added Managed Service Identity support for egress (#1884)
+- Added Process.RuntimeId token for collection rule action properties (#1870)
```
[the remark this answers is not in the document]
Comment (jander-msft) [reply documentation/releaseNotes/releaseNotes.md:10]: I think renaming to RuntimeId makes sense; we were already using it for process filter: https://github.com/dotnet/dotnet-monitor/blob/main/src/Microsoft.Diagnostics.Monitoring.WebApi/DiagProcessFilter.cs#L144

**Review by wiktork — APPROVED** [review]

**Pull request merged**